test/parallel/test-net-socket-destroy-twice.js

```diff
@@ -23,7 +23,7 @@
 const common = require('../common');
 const net = require('net');
 
-const conn = net.createConnection(common.PORT);
+const conn = net.createConnection(0);
```

Comment:
(Which it looks like this one is expected to fail because there's nothing listening on the port. But then the issue is: Does this test still test the thing it was written to test? I don't know the answer to that.)